client/landing/gutenboarding/onboarding-block/design-selector/design-card.tsx

```diff
@@ -10,15 +10,14 @@ import classnames from 'classnames';
  */
 import { Card } from '../../components/card';
 import { CardMedia } from '../../components/card/media';
-import { Template } from '../../stores/verticals-templates/types';
 
 const gridWidth = 960;
 const srcSet = ( src: string, widths: number[] ) =>
 	widths.map( width => addQueryArgs( src, { w: width } ) + ` ${ width }w` ).join( ', ' );
 
 interface Props {
 	isSelected?: boolean;
-	design: Template;
+	design: import('@automattic/data-stores').VerticalsTemplates.Template;
```

Comment:
I find inlining these a bit ugly :grimacing: Bit curious about the rationale -- b/c it's TypeScript only, thus lost in transpilation?

Reply:
That's is the rationale, but we should do some testing. With the package being side-effect free, I'd expect the result to be the same if we use a standard import.

With the inline type import, I'm highly confident there's no impact on the build. With standard esm import, I'm not as confident.